Grbl_Esp32/Custom/atari_1020.cpp

```diff
@@ -165,7 +167,7 @@ void atari_home_task(void* pvParameters) {
                 if (homing_attempt > ATARI_HOMING_ATTEMPTS) {
                     // try all positions plus 1
                     grbl_msg_sendf(CLIENT_SERIAL, MSG_LEVEL_INFO, "Atari homing failed");
-                    inputBuffer.push("G90\r");
+                    WebUI::inputBuffer.push("G90\r");
```

Comment:
as raised in slack:
inputBuffer is not part of webUI but Macro : https://github.com/bdring/Grbl_Esp32/wiki/Macro-Button-Feature

so I think put it in WebUI namespace could be confusing